notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<blockquote>galaxy, they</blockquote><p>Change comma to a semi-colon.</p><blockquote>is called <strong>3c273</strong>, it is one</blockquote><p>Split into 2 sentences?</p><p> </p><p>More generally, can we state a science case for what the user of the tutorial might be doing? For example, we could preface why someone would want to make a historical coverage plot. Namely, that we can learn different information from different wavelengths of light - I'm not the expert on quasars but maybe a brief summary of what kinds of conclusions or measurements can be made from different wavelength ranges. Then for the time aspect, maybe a brief explanation of how quasars could be time-variable. (My extremely surface-level research from https://astrobites.org/2017/07/11/extreme-variability-quasars/ led me to realize this.) So a person would want to know the wavelength vs. time coverage so that they could study the emission from the quasar over time. I can tell this is what you're getting at with "the history of this quasar" but it could be stated more clearly.</p><p> </p><p>Can you elaborate more on what "draw our own conclusions" means?</p><p> </p><p>One more reminder, the updated template (https://sciencebetter.slack.com/archives/C03FDR44L07/p1660233388139449) includes a workflow section that is basically a Table of Contents. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>